src/app/modules/main/wallet_section/transactions/controller.nim

```diff
@@ -97,8 +97,8 @@ proc suggestedFees*(self: Controller, chainId: int): string =
   let suggestedFees = self.transactionService.suggestedFees(chainId)
   return suggestedFees.toJson()
 
-proc suggestedRoutes*(self: Controller, account: string, amount: float64, token: string): string = 
-  let suggestedRoutes = self.transactionService.suggestedRoutes(account, amount, token)
+proc suggestedRoutes*(self: Controller, account: string, amount: float64, token: string, disabledNetworks: seq[string]): string =
```

Comment:
as mentioned in status-go pr, it will be better to use disabledChainIds: seq[int]